api/app/Http/Controllers/AuthController.php

```diff
@@ -183,7 +182,7 @@ public function handleProviderCallback($provider, Socialite $socialite, UserRepo
             // The app is connected with the service, but the 3rd party service
             // is not configured or allowed to return email addresses, so we
             // can't process the data further. Let's throw an exception.
-            Log::critical('Provider '.$provider.' does not return email.');
+            \Log::critical('Provider '.$provider.' does not return email.');
```

Comment:
We can do one better than that for facades - https://github.com/barryvdh/laravel-ide-helper over the weekend added support for Lumen. I'll be implementing that shortly.